recipe/meta.yaml

```diff
@@ -1,54 +1,47 @@
-{% set name = "kombu" %}
 {% set version = "4.6.7" %}
-{% set sha256 = "67b32ccb6fea030f8799f8fd50dd08e03a4b99464ebc4952d71d8747b1a52ad1" %}
-{% set build = 0 %}
 
 package:
-    name: {{ name|lower }}
-    version: {{ version }}
+  name: kombu
+  version: {{ version }}
 
 source:
-    fn: {{ name }}-{{ version }}.tar.gz
-    url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/k/kombu/kombu-{{ version }}.tar.gz
+  sha256: 67b32ccb6fea030f8799f8fd50dd08e03a4b99464ebc4952d71d8747b1a52ad1
 
 build:
-    noarch: python
-    number: {{ build }}
-    script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-    build:
-        - python
-        - pip
-
-    run:
-        - python
-        - simplejson
-        - sqlalchemy
-        - amqp >=2.5.2,<3.0
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - importlib_metadata >=0.18  # [py<=37]
+    - amqp >=2.5.2,<2.6
```

Comment:
@kwilcox I'm following the package metadata here but do you have reasons to keep `sqlalchemy` and `amqp >=2.5.2,<3.0` instead of `<2.6`?

Reply:
Nope, its `<2.6` in the source repo so that should be carried over here